index.html

```diff
@@ -9,31 +9,31 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
-    <title>damdafayton Portfolio</title>
+    <title>damdafayton Portfolio Page</title>
 </head>
 
 <body>
-    <nav>
+    <header>
         <h1>Welcome</h1>
-        <div id="burger-container">
+        <nav id="burger-container">
             <div class="burger-line"></div>
             <div class="burger-line"></div>
             <div class="burger-line"></div>
-        </div>
-    </nav>
-    <article>
+        </nav>
+    </header>
+    <section>
         <h1><span>Hey there, I’m "damdafayton".
             </span><span> I’m a software developer.</span></h1>
-        <section>I can help you build a product , feature or website Look through some of my work and experience! If
+        <div>I can help you build a product , feature or website Look through some of my work and experience! If
```

Comment:
I am so sorry about how you feel. Please note that your design is very much okay but it must be exactly like the design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). Including the social media icons.

### Figma design with what you need to include in your design:
![figma](https://user-images.githubusercontent.com/10332499/148352593-298ad284-5366-4a0d-a74f-d207eeded779.PNG)

### Your design:
![your-design](https://user-images.githubusercontent.com/10332499/148352649-27429a15-b65a-477b-9e3a-67730f7851be.PNG)

Please ensure the design is the same as the Figma because having a nice design increases your chances for a job as recruiters will be checking your GitHub projects.

Reply:
Can you send me the link of the file please. You are checking the wrong commit.
This is the latest update:
https://github.com/backtofayton/portfolio/blob/mobile-phase1/index.html

Also can you explain me how to include dozens of vectors as background image please? There is no source for these images or vectors on Figma.

Reply:
@backtofayton 
Did you open another pull request?

Reply:
Why should I open another pull request?
Every review has this warning including yours:
```
Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
```

Reply:
I cant get any image from Figma. I have written you on Slack.

Reply:
Our conversation on Slack

![slack](https://user-images.githubusercontent.com/83017311/148777603-a319f875-6917-4420-806a-bb089163c639.jpg)